src/main/java/fi/aalto/cs/apluscourses/ui/AboutDialog.java

```diff
@@ -1,21 +1,26 @@
 package fi.aalto.cs.apluscourses.ui;
 
+import static fi.aalto.cs.apluscourses.utils.PluginResourceBundle.getAndReplaceText;
 import static fi.aalto.cs.apluscourses.utils.PluginResourceBundle.getText;
 
 import com.intellij.icons.AllIcons;
 import com.intellij.ide.BrowserUtil;
 import com.intellij.ui.components.JBLabel;
 import com.intellij.ui.components.labels.LinkLabel;
-import javax.swing.Box;
-import javax.swing.BoxLayout;
-import javax.swing.JOptionPane;
-import javax.swing.JPanel;
-import javax.swing.SwingConstants;
+
+import javax.swing.*;
+
+import icons.PluginIcons;
+
+import java.io.IOException;
+import java.util.Properties;
 import org.jetbrains.annotations.NotNull;
 
 public class AboutDialog {
 
   private static final String A_COURSES_PLUGIN_PAGE = "https://plugins.jetbrains.com/plugin/13634-a-courses";
+  private static final String A_PLUS_PAGE = "https://plus.cs.aalto.fi/";
+  private static final String GITHUB_PAGE = "https://github.com/Aalto-LeTech/intellij-plugin";
```

Comment:
now when there's more of them, I wonder, if we should also externalize the URLs around the code into the `resources.properties` (as one of the features of Java props is that you can replace them without recompiling the code - not very useful for us but say Otto or some other teacher might really benefit from this when testing)? @Aalto-LeTech/idea-plugin-dev

Reply:
I agree we should. Perhaps not in the same resource file though with the UI strings (also because strings can be localized in the future).

Reply:
I am lightly in favor of the status quo (keeping them in the code), but I don't have any strong opinions this

Reply:
In any case, there's no need to address that in this PR. This is fine for now.